tests/FirstTest.php

```diff
@@ -113,17 +113,6 @@ public function testCancelPendingStreamWillReject()
         $this->expectPromiseReject($promise);
     }
 
-    public function testNoGarbageCollectionCyclesAfterClosingStream()
-    {
-        \gc_collect_cycles();
-        $stream = new ThroughStream();
-        $promise = Stream\first($stream);
-
-        $stream->close();
-
-        $this->assertSame(0, \gc_collect_cycles());
-    }
-
```

Comment:
Note that this removed test is somewhat similar to the test 2 functions below (both added via #36 recently). The only difference here is that this one does not handle the (expected) promise rejection, hence why this one reports an unhandled rejection (via https://github.com/reactphp/promise/pull/248). It's easy to fix this by adding a promise rejection handler, but this would make it identical to the other test, hence why I've decided to simply remove this.